src/dewrapper/dewrapper.c

```diff
@@ -258,7 +262,14 @@ static int depth_engine_thread(void *param)
                                                     &outputCaptureInfo,
                                                     NULL);
             tickcounter_get_current_ms(dewrapper->tick, &stop_time);
-            if (deresult != K4A_DEPTH_ENGINE_RESULT_SUCCEEDED)
+            if (deresult == K4A_DEPTH_ENGINE_RESULT_FATAL_ERROR_WAIT_PROCESSING_COMPLETE_FAILED ||
+                deresult == K4A_DEPTH_ENGINE_RESULT_FATAL_ERROR_GPU_TIMEOUT)
+            {
+                LOG_ERROR("Timeout during depth engine process frame. This could indicate GPU is hung.", 0);
```

Comment:
FYI, this is an implementation detail in the public space. you might just want to say the GPU is in a bad state, and we must restart.

Reply:
Yeah, I know. I tried to make the log message helpful without going too much into the details. Do you have suggestion on how to change this?

Reply:
I changed the error message to your suggestion.